docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,484 @@
+                "readOnly":false
+            }
+        ]
+    }
+```
+
+Now apply it:
+
+```shell
+kubectl apply -f datasources.yaml
+```
+
+## Install kube prometheus stack
+
+This will install:
+
+- Prometheus
+- Prometheus Configuration
+- Grafana & default dashboards
+
+Save this file as `values.yaml`:
+
+```yaml
+grafana:
+  adminPassword: admin
+  sidecar.datasources.defaultDatasourceEnabled: false
+prometheus:
+  prometheusSpec:
+    additionalScrapeConfigs:
+      - job_name: "scrape_klt"
+        scrape_interval: 5s
+        static_configs:
+          - targets: ['keptn-klt-lifecycle-operator-metrics-service.keptn-lifecycle-toolkit-system.svc.cluster.local:2222']
+```
+
+```shell
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm upgrade --install observability-stack prometheus-community/kube-prometheus-stack --version 48.1.1 --namespace monitoring --values=values.yaml --wait
+```
+
+## Access Grafana
+
+```shell
+kubectl -n monitoring port-forward svc/observability-stack-grafana 80
+```
+
+- Grafana username: `admin`
+- Grafana password: `admin`
+
+View the Keptn dashboards at: `http://localhost/dashboards`
+
+Remember that Jaeger and Grafana weren't installed during the first deployment
+so expect the dashboards to look a little empty.
+
+## Deploy v0.0.2 and populate Grafana
+
+By triggering a new deployment, Keptn will track this deployment and the Grafana dashboards will actually have data.
+
+Modify your `app.yaml` and change the `app.kubernetes.io/version` from `0.0.1` to `0.0.2`.
+
+Apply your update:
+
+```shell
+kubectl apply -f app.yaml
+```
+
+After about 30 seconds you should now see two `keptnappversions`:
+
+```shell
+kubectl -n keptndemo get keptnappversion
+```
+
+Expected output:
+
+```shell
+NAME                          APPNAME        VERSION   PHASE
+keptndemoapp-0.0.1-6b86b273   keptndemoapp   0.0.1     Completed
+keptndemoapp-0.0.2-d4735e3a   keptndemoapp   0.0.2     AppDeploy
+```
+
+Wait until the `PHASE` of `keptndemoapp-0.0.2` is `Completed`.
+This signals that the deployment was successful and the pod is running.
+
+View the Keptn Applications Dashboard and you should see the DORA metrics and an OpenTelemetry per trace:
+
+![keptn applications dashboard](../assets/keptnapplications.png)
+
+![deployment trace](../assets/deploymenttrace.png)
+
+## More control over KeptnApp
+
+You may have noticed that the `KeptnApp` Custom Resources are created automatically by KLT.
+
+The lifecycle toolkit automatically groups workloads into `KeptnApp` by looking for matching `part-of` annotations.
+Any workloads with the same `part-of` annotation is said to be `part-of` the same `KeptnApp`.
+
+However, you can override this automatic behaviour by creating a custom `KeptnApp` CRD.
+In this way, you are in full control of what constitutes a Keptn Application.
+See [Define a Keptn Application](../implementing/integrate/#define-keptnapp-manually) for more information.
```

Comment:
this link here seems to be broken in the deploy preview... i don't know why. maybe the path is not right?